docs/source/sft_trainer.mdx

```diff
@@ -410,6 +410,61 @@ We have tested NEFTune by training `mistralai/Mistral-7B-v0.1` on the [OpenAssis
 </div>
 
 Note however, that the amount of performance gain is _dataset dependent_ and in particular, applying NEFTune on synthetic datasets like [UltraChat](https://huggingface.co/datasets/stingning/ultrachat) typically produces smaller gains.
+
+### Accelerate fine-tuning using `unsloth` library
```

Comment:
### Accelerate fine-tuning using `unsloth` library
maybe to

### Accelerate fine-tuning using `unsloth`